### ISSUE TITLE
Others ways to move passes in new/different topics (tabs/groups)

### ISSUE THREAD

**Issue by theroggy** [ligi/PassAndroid#134]
Might be a personal problem, but I don't think swiping a pass is a very intuitive way to move it to another topic. I really had to search for it for several minutes before I found it: I was afraid when I would swipe it would be deleted. 

I think adding some additional ways to do this would be a good improvement, eg.: 
- a "right-click" (long press) option on the pass to get to the "move it to a different topic" option
- an option to move it/set a pass to another topic when you are editing a pass: you can change the category/icon/... now but not the topic

Btw. Maybe because I'm  not a native english speake but I think eg. "Group" would sound clearer than "topic".

**Comment by simonspa** [comment]
I support this. I'm having problems moving passes and until today I have not figured out when a pass is moved and when I simply slide and change the topic, it's completely unintuitive to me...

**Comment by janvonde** [comment]
Yes, swiping means deleting IMHO as well. long press and a menu then would be my preferred way as well.

**Comment by jfml** [comment]
This is the only thing I really dislike about this otherwise amazing app. It's probably me being old and weird but it takes me about 15 swipe-tries until one actually works and does what it's supposed to do, so having another way (eg long press) would be amazing.

**Comment by alsternerd** [comment]
The swiping only annoys me to date, though it's the most decent app out there.
The swiping makes me want to scream everytime, because it breaks my usage experience. Every other app doesn't do that, I can at least just long click and then use the popup menu.
I just want to long press and then select delete, move or other, that'd be great.
